sdk/maps/maps-timezone-rest/src/mapsTimezone.ts

```diff
@@ -0,0 +1,117 @@
+
```

Comment:
1. Remove empty line
2. Please rename this file to `MapsTimezone.ts` for consistency